### PR TITLE
`model.config` handle breaking change in hub

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

**Review by osanseviero — APPROVED** [review]
LGTM Once the internal PR is merged

**Comment by julien-c** [comment]
the internal PR should be deployed in the next hour or so

**Review by osanseviero — APPROVED** [review]
This is now deployed :partying_face: https://huggingface.co/api/models/dfurman/Mixtral-8x7B-Instruct-v0.1?library=true

**Pull request merged**
